uw_sws/models.py

```diff
@@ -377,9 +377,11 @@ def json_data(self):
         if self.start_date:
             data["start_date"] = self.start_date.strftime("%Y-%m-%d %H:%M")
             data["end_date"] = self.end_date.strftime("%Y-%m-%d %H:%M")
+        if self.building:
             data["building"] = self.building
+        if self.room_number:
             data["room_number"] = self.room_number
-
+            data["room"] = self.room_number
```

Comment:
I don't understand what the room attribute is for since it's just a copy of room_number?